korean/instagram/insta.md

```diff
@@ -0,0 +1,14 @@
+인스타그램에 인증샷을 올린다. 하트 숫자가 늘어나는 것을 보고 기분이 좋아졌다.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.